File: DPR.cpp
#include <iostream>
#include <string>

using namespace std;

class DPR
{
    private: 
        int id;
        string nama;
        string bidang;
        string partai;

    public:
        DPR()
        {
            this->id = 0;
            this->nama = "";
            this->bidang = "";
            this->partai = "";
        }

        DPR(int id, string nama, string bidang, string partai)
        {
            this->id = id;
            this->nama = nama;
            this->bidang = bidang;
            this->partai = partai;
        }

        int get_id()
        {
            return this->id;
        }

        void set_id(int id)
        {
            this->id = id;
        }

        string get_nama()
        {
            return this->nama;
        }

        void set_nama(string nama)
        {
            this->nama = nama;
        }

        string get_bidang()
        {
            return this->bidang;
        }

        void set_bidang(string bidang)
        {
            this->bidang = bidang;
        }

        string get_partai()
        {
            return this->partai;
        }

        void set_partai(string partai)
        {
            this->partai = partai;
        }

        ~DPR()
        {

        }

};

File: PHP/DPR.php
<?php
class DPR {
    private $id;
    private $nama;
    private $bidang;
    private $partai;
    private $foto;

    public function __construct($id, $nama, $bidang, $partai, $foto) {
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->foto = $foto;

    }

    public function get_id() {
        return $this->id;
    }

    public function get_nama() {
        return $this->nama;
    }

    public function get_bidang() {
        return $this->bidang;
    }

    public function get_partai() {
        return $this->partai;
    }

    public function set_nama($nama) {
        $this->nama = $nama;
    }

    public function set_bidang($bidang) {
        $this->bidang = $bidang;
    }

    public function set_partai($partai) {
        $this->partai = $partai;
    }

     public function set_foto($foto){
        $this->foto = $foto;
    }
    public function get_foto(){
        return $this->foto;
    }
}

?>